src/components/PlayerDash.jsx

```diff
@@ -15,38 +17,47 @@ export const PlayerDash = React.createClass({
   getInitialState() {
     return {
       players: [],
+      graphdata: [],
       loaded: false
     }
   },
 
   componentWillMount() {
+    this.firebase = new FirebaseLib();
     // We are likely to want to read all player data e.g. top adversary, last 10 matches etc
     // So leaving this here for now. Woould be nicer to have a store where this data is kept without requesting again
-    let fbPath = [conf.firebaseUrl, 'players'].join('/');
-    this.fireBase = new Firebase(fbPath);
     this.loadData(); // should update to bindAsObject/Array
   },
 
   render() {
     // console.log(this.state.players);
     const player = _.find(this.state.players, p => p.id == this.props.params.playerId);
+    const thedata = this.state.graphdata;
+
     return (
       <div className="Player">
       { player &&
-          <div className="col-md-3">
-            <PlayerCard {...player} />
-          </div>
+        <div className="col-md-3">
+        <PlayerCard {...player} />
+        </div>
       }
       { !player &&
-          "Player not found."
+        "Player loading..."
+      }
+      { thedata &&
+        <div className="col-md-9">
+        <EloGraph graph={thedata} />
+        </div>
+      }
+      { !thedata &&
+        "Graph loading..."
       }
       </div>
-    );
+      );
   },
 
   loadData() {
-
```

Comment:
I think loadData should be separated into 2 functions. One for loadPlayers, one for loadPlayerGames or similar.  It's just getting a bit long as it is and I think since nothing is shared between the two processes it makes sense to separate them.